apps/portal-front/src/components/service/trial-instances/start-trial-button.tsx

```diff
@@ -51,7 +51,18 @@ export const StartTrialButton: React.FC<Props> = ({
   const t = useTranslations();
   const environment = useRelayEnvironment();
 
-  const { freeTrial } = useFreeTrial();
+  const { freeTrial, isBlacklisted } = useFreeTrial();
+
+  if (isBlacklisted) {
```

Comment:
The condition is reversed: button must be displayed when user is not blacklisted